docs/dev_environment_setup.md

```diff
@@ -116,9 +116,9 @@ The [`goimports`](https://pkg.go.dev/golang.org/x/tools/cmd/goimports) tool is u
 
 ## Node.js/npm
 
-We currently support Node.js v16 for this repo; Node 17 support is currently blocked by [this `create-react-app` issue](https://github.com/facebook/create-react-app/issues/11562). 
+We currently support Node.js v16 for this repo; Node 17 support is currently blocked by [this `create-react-app` issue](https://github.com/facebook/create-react-app/issues/11562).
 
-The easiest way to install this specific version of Node/npm is to use [`nvm`](https://github.com/nvm-sh/nvm). To install `nvm`, run 
+The easiest way to install this specific version of Node/npm is to use [`nvm`](https://github.com/nvm-sh/nvm). To install `nvm`, run
```

Comment:
If we're updating node version, I think it's worth:
- Introducing an `.nvmrc` into the root
- Modifying `Dockerfile.client` and `Dockerfile.client_ci` to match

Thoughts?